charts/stable/homarr/Chart.yaml

```
@@ -18,7 +18,7 @@ name: homarr
 sources:
   - https://github.com/truecharts/charts/tree/master/charts/stable/homarr
   - https://github.com/ajnart/homarr
-version: 6.0.12
+version: 6.0.13
```

```suggestion
version: 7.0.0
```
If upstream is marked as breaking, we should too. Regardless of "it might not break"